scripts/run.sh

```
@@ -5,17 +5,21 @@ mkdir -p /appdata/db
-        echo "ERROR: DB migrations failed, aborting startup"
-        exit 1
-    fi
-fi
+run_migrations() {
+  if [ "${DB_MIGRATIONS_DISABLED:-false}" = "true" ]; then
+    echo "DB migrations are disabled, skipping"
+    return
+  fi
+
+  echo "Running DB migrations"
+  DISABLE_REDIS_LOGS=true node ./db/migrations/$DB_DIALECT/migrate.cjs ./db/migrations/$DB_DIALECT
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)